sdetools/sdelib/conf_mgr.py

```
@@ -423,6 +425,20 @@ def process_json_str_dict(self, key):
         except Exception, err:
             raise UsageError('Unable to process %s (not a JSON dictionary). Reason: %s' % (key, str(err)))
 
+    def process_json_dict(self, key):
+        try:
+            if not self[key]:
```

I think I do want to let the exception propagate here - I'll check again